README.md

```diff
@@ -1,23 +1,29 @@
 # skonfig
 
-This is [skonfig](https://skonfig.li), a fork of [ungleich](https://github.com/ungleich)'s [cdist](https://github.com/ungleich/cdist) (after [f061fb1](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
+This is [skonfig](https://skonfig.li), a streamlined and compartmentalized version of
+[ungleich](https://github.com/ungleich)'s [cdist](https://github.com/ungleich/cdist)
+(forked after [f061fb1](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
 
 Work is mostly split between three main repositories:
 
 * **skonfig** - implementation of the **skonfig tool** and quick **getting started** bits,
 * [base](https://github.com/skonfig/base) - **base** explorers and types,
 * [extra](https://github.com/skonfig/extra) - **extra** types (community contributions).
- 
+
+Our chat room is `#skonfig:matrix.org` (you need [Matrix client](https://matrix.org/docs/projects/try-matrix-now/)).
+
 ## Getting Started
 
 There will be no versioned releases fow now - everything is expected to be used from the repositories.
 
-We are currently targeting existing `cdist` power users, so expect some rough edges.
+We are currently targeting (`cdist`) power users, so expect some rough edges.
 
 ### Step 1: Clone repositories
 
 ```sh
-cd ~/Where/YOU/keep/your_repos/
+cd ~/Where/YOU/keep/your_repos
+mkdir skonfig
+cd skonfig
 git clone https://github.com/skonfig/skonfig
 git clone https://github.com/skonfig/base
```

Comment:
Dunno, I prefer keeping repos in one place and `skonfig` figuring all out.
But for some users this might make more sense. Especially if we install `skonfig` with `pip`?
So let's keep both ways supported, but recommend cloning into `~/.skonfig/set/`?